dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -316,6 +316,7 @@
 			reg = <0x40020000 0x400>;
 			interrupts = <11 0 12 0 13 0 14 0 15 0 16 0 17 0>;
 			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x1>;
+			st,mem2mem;
```

Comment:
Does it apply to v2?

Reply:
yes, meaning is defined by linux: support of mem2mem